src/archive.rs

```diff
@@ -9,8 +9,19 @@ use std::fs::File;
 use std::io;
 use std::string::String;
 
-const MIN_SUPPORTED_VERSION: Version = (1, 10, 0);
-const MAX_SUPPORTED_VERSION: Version = (1, 15, 0);
+// Historical version numbers are described in `postgres/src/bin/pg_dump/pg_backup_archiver.h`
+
+/// add tablespace
+pub const K_VERS_1_10: Version = (1, 10, 0);
+
+/// add tableam
+pub const K_VERS_1_14: Version = (1, 14, 0);
+
+/// add compression_algorithm in header
+pub const K_VERS_1_15: Version = (1, 15, 0);
+
+/// BLOB METADATA entries and multiple BLOBS, relkind
+pub const K_VERS_1_16: Version = (1, 16, 0);
```

Comment:
Is there a need to make these fully public? Perhaps `pub(crate)` is a better fit here.

Reply:
In my opinion it could be either way.

Theoretically, someone outside might want to compare what dump version they have and whether some fields will not be empty because of it.

I will do as you wish.

Reply:
@wichert I am waiting for your decision.